events/janus_rabbitmqevh.c

```diff
@@ -0,0 +1,724 @@
+/*! \file   janus_rabbitmqevh.c
+ * \author Lorenzo Miniero <lorenzo@meetecho.com>
```

Comment:
Should be:

    \author Piter Konstantinov <your mail>

as it's really your plugin!